.github/workflows/iroha2-dev-pr.yml

```diff
@@ -18,10 +18,9 @@ concurrency:
 
 env:
   CARGO_TERM_COLOR: always
-  CARGO_INCREMENTAL: 0
```

Comment:
`CARGO_INCREMENTAL=0` is implied by [`Swatinem/rust-cache`](https://github.com/Swatinem/rust-cache)